DOCS-3301 add background event tracking documentation (#840)
DataDog/dd-sdk-ios: docs/rum_collection/_index.md

{{% /tab %}}
{{< /tabs >}}

### Track background events

> To track events (crashes, network requests etc.) when your application is in the background, i.e. there is no active view, add the following snippet during initialization while setting up Datadog configuration:

Reviewer: ```suggestion
You can track events such as crashes and network requests when your application is in the background (for example, no active view is available). 

Add the following snippet during initialization in your Datadog configuration:
```

{{< tabs >}}
{{% tab "Swift" %}}
```swift
.trackBackgroundEvents()

```

> <div class="alert alert-info"><p>Note that tracking background events could lead to additional sessions that might impact billing. For questions contact support@datadoghq.com</p>

Reviewer: ```suggestion
<div class="alert alert-info"><p>Tracking background events may lead to additional sessions, which can impact billing. For questions, <a href="https://docs.datadoghq.com/help/">contact Datadog support.</a></p>
```
